samples/subsys/ipc/rpmsg_multi_instance/remote/prj.conf

```diff
@@ -0,0 +1,32 @@
+CONFIG_STDOUT_CONSOLE=n
+CONFIG_PRINTK=n
+CONFIG_IPM=y
+CONFIG_HEAP_MEM_POOL_SIZE=4096
+CONFIG_SEGGER_RTT_BUFFER_SIZE_UP=4096
+
+# Enable Rpmsg multiple instance
+CONFIG_RPMSG_MULTI_INSTANCE=y
+CONFIG_RPMSG_MULTI_INSTANCE_REMOTE=y
+CONFIG_RPMSG_MULTI_INSTANCES_NO=2
+
+CONFIG_IPM=y
+CONFIG_IPM_NRFX=y
```

Comment:
And here as well we have SoC-specific entries.

Reply:
Modified